src/components/GraphingPresentationJob.js

```diff
@@ -43,9 +43,11 @@ class PieRechartReport extends React.Component {
     ).studentNames;
 
     return {
-      text: studentNames.map((studentName) => {
-        console.log(studentName);
-        return `${studentName}<br/>`;
+      text: studentNames.sort().map((studentName, index) => {
+        if (index % 6 === 0) {
+          return ` ${studentName}<br/>`;
```

Comment:
@komalng What does it mean? Are we adding line breaks after every 6th student? If yes, why?

Reply:
This code is already merged in the main branch. The reason to write a break tag for every 6th student is that it will give data in a structural way means 6 students' data will show in a line for 7th student data will come to the next line.